tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -1120,6 +1132,46 @@ static void test_scanx_main(void)
 	}
 	printk("success.\n");
 
+	printk("Waiting for Periodic Advertising Report of 0 bytes...");
+	while (!is_sync_report) {
+		k_sleep(K_MSEC(100));
+	}
+	printk("done.\n");
+
+	printk("sync_report_len = %u\n", sync_report_len);
+
+	if (sync_report_len != 0) {
+		FAIL("Incorrect Periodic Advertising Report data.");
+	}
+
+	printk("Waiting for Periodic Advertising Report of %u bytes...",
+	       sizeof(per_adv_data1));
```

Comment:
per_adv_data1 (and per_adv_data2) are undefined. Is there a dependency on another PR?

Reply:
yes, depends on #29778